### DISCUSSION TITLE
Loading indicator from `loading.tsx` not showing up on dynamic routing `/category/[...props]/page.tsx`

### DISCUSSION THREAD

**Discussion by vildantursic** [vercel/next.js#48496] [Help]
### Summary

Hi,

I have page where number of params is dynamic (e.g. `category/vehicle/audi` => `category/vehicle/audi/a4` =>  `category/vehicle/audi/a4/2014`)
and with current implementation `/category/[...props]/page.tsx` loading indicator from `loading.tsx` is triggered only once when I enter categories page and every subsequent page load/path loading does not show up.

I did try with `/category/[props]/page.tsx` and loading works fine but my path would have to be (e.g. `category/vechicle-audi-a4-2014`) which is not really convenient.

How can I make loading work with dynamic props on `/category/[...props]/page.tsx`?

**Comment by MAhmedSid** [comment]
I am also getting issue, but i am bit confuse that in my app , loading.tsx file not rendering when i fetch data in serverside pages, even i have tried to make loading.tsx as client side but also not working for me .

Next js v13.4
